README.md

```diff
@@ -26,46 +26,44 @@ This build plugin is a utility for supporting Next.js on Netlify. To enable serv
 
 ## Installation and Configuration
 
-There are two ways to install the plugin in your Next.js on Netlify site: with the Netlify UI or with file-based installation.
+As of v3.0.0, all new sites deployed to Netlify with Next.js will automatically build with this plugin for a seamless experience deploying Next.js on Netlify!
 
-**UI-based Installation**
+This means that you don't have to do anything — just build and deploy your site to Netlify as usual and we'll handle the rest.
 
-You can install this plugin using this [direct in-app installation link](http://app.netlify.com/plugins/@netlify/plugin-nextjs/install) or from your team's [Plugins directory](https://app.netlify.com/plugins) in the Netlify UI.
+You're able to remove the plugin at any time by visiting the Plugins tab on your site.
```

Comment:
@verythorough @lindsaylevine can you please check the updated installation instructions to make sure I didn't miss anything?

Reply:
I think @rstavchansky would be the better authority on giving feedback on instructions.

Reply:
yep, I'm on it. I had this wrong in my head